app/js/components/orderEntry/ActiveOrders.jsx

```diff
@@ -64,4 +100,22 @@ class ActiveOrders extends React.Component {
   }
 }
 
-export default ActiveOrders;
+ActiveOrders.propTypes = {
+  activeOrderAction: PropTypes.func.isRequired,
+  drugOrder: PropTypes.shape({
+    loading: PropTypes.shape({}).isRequired,
```

Comment:
Correct me if I'm wrong but I think the loading prop is of type boolean

Reply:
Yes, working on this